CorsixTH/Lua/objects/plant.lua

```diff
@@ -70,11 +70,13 @@ object.orientations = {
 -- For litter: put broom back 356
 -- take broom out: 1874
 -- swoop: 1878
--- For plant: droop down: 1950
--- back up again: 1952
+-- Frames for plant states are
```

Comment:
No it is not an index, as it starts at 0. It also ends at 5, so there is a missing index somewhere.
I don't want to debug and fix all plant-related stuff now.